st2api/st2api/controllers/root.py

```diff
@@ -14,20 +14,40 @@
 # limitations under the License.
 
 from pecan import expose
+from pecan.core import redirect
+from oslo.config import cfg
 
 from st2common import __version__
 from st2common import log as logging
 import st2api.controllers.v1.root as v1_root
 
+__all__ = [
+    'WebUIRootController',
+    'RootController'
+]
+
 LOG = logging.getLogger(__name__)
 
 
+class WebUIRootController(object):
+    @expose(generic=True)
+    def index(self):
+        return redirect(location='/webui/index.html')
```

Comment:
To clarify it - redirect is there because of the way static middleware works - there is no support for index files and directory listings, so without a redirecting, HTTP server would previously return 404 not found.

So this is basically just a workaround (we also can't and dont want to use "template" functionality here - templates are in a different directory) :)